openpype/pipeline/workfile/workfile_template_builder.py

```diff
@@ -1814,3 +1838,13 @@ def get_errors(self):
 
     def create_failed(self, creator_data):
         self._failed_created_publish_instances.append(creator_data)
+
+
+def get_libraries_project_names():
```

Comment:
Double plural is not needed.
```suggestion
def get_library_project_names():
```